packages/css-data/src/property-parsers/box-shadow-property-extractor.test.ts

```diff
@@ -0,0 +1,125 @@
+import { describe, expect, test } from "@jest/globals";
+import { extractBoxShadowProperties } from "./box-shadow-property-extractor";
+
+describe("extractBoxShadowProperties", () => {
+  test("parses and extracts values from 5em 5em 2em 2em #0000", () => {
+    const { inset, blur, spread, color } = extractBoxShadowProperties({
```

Comment:
This is more like parser and that thing producing tuple is tokenizer

Reply:
Should i change the name to `tokenizeBoxShadowTuple` or something else ?